package.json

```diff
@@ -62,7 +62,7 @@
     "chai": "^3.5.0",
     "cookie-parser": "^1.3.4",
     "es5-shim": "^4.1.0",
-    "eslint-config-loopback": "^1.0.0",
+    "eslint-config-loopback": "^4.0.0",
```

Comment:
We should add:

```
"lint": "eslint .",
"posttest": "npm run lint"
```

But please do this in a separate PR, this one is huge as is already.